Text/Link Joker.txt

```diff
@@ -966,7 +966,7 @@ global.UnitGrade[CardStat] = 2
 global.CardInClan[CardStat] = 24
 }
 
-CardStat = x+5
+CardStat = 2223
```

Comment:
This was supposed to be left as-is, you only had to remove most of the data and add a parameter `global.CardReassignedId[CardStat] = 2223`. That's what I meant by "reassign", not literally reassigning the variable.

Reply:
Completely forgot about the CardReassign method. Will keep it in mind shoukd it come up in the future. Apologies for the inconvenience.